heron/uploaders/src/java/com/twitter/heron/uploader/hdfs/HdfsUploader.java

```diff
@@ -65,33 +66,36 @@ protected boolean isLocalFileExists(String file) {
   }
 
   @Override
-  public URI uploadPackage() {
+  public URI uploadPackage() throws UploaderException {
     // first, check if the topology package exists
     if (!isLocalFileExists(topologyPackageLocation)) {
-      LOG.info("Topology file " + topologyPackageLocation + " does not exist.");
-      return null;
+      throw new UploaderException(
+        String.format("Expected topology package file to be uploaded does not exist at '%s'",
+            topologyPackageLocation));
     }
 
     // if the dest directory does not exist, create it.
     if (!controller.exists(destTopologyDirectoryURI)) {
-      LOG.info("The destination directory does not exist; creating it.");
+      LOG.info("The destination directory does not exist. Creating it now.");
```

Comment:
hmm, I don't see it. I'm referring to the Log.info above. The creating it now message.